### PR TITLE
fix bug -- The Compose text and Messages should be hidden if the user doesnt SMS…

### PR DESCRIPTION
**Pull request by caffffe**

### CODE REVIEW

File: dev-server/containers/MainView/index.js [hunk 1]
```diff
@@ -61,6 +61,17 @@ const MainView = connect((state, props) => {
   if (messageTab) {
     messageTab.noticeCounts = props.messageStore.unreadCounts;
   }
+  if (props.rolesAndPermissions.permissions &&
+      props.rolesAndPermissions.permissions.OutboundSMS !== true &&
+      props.rolesAndPermissions.permissions.InternalSMS !== true
```
Comment (embbnux) [inline dev-server/containers/MainView/index.js]: hidden when permissions.ReadMessages is not true?

Comment (caffffe) [inline dev-server/containers/MainView/index.js]: Do you mean when a user has SMS/MMS/pager permission and no readmessages permission,hide messages page but show composeText page？

Comment (embbnux) [inline dev-server/containers/MainView/index.js]: yeah

File: dev-server/containers/MainView/index.js [hunk 2]
```diff
@@ -61,6 +61,25 @@ const MainView = connect((state, props) => {
...
+      ) {
+    if (props.rolesAndPermissions.permissions.ReadMessages === true) {
+      return {
+        tabs: tabs.filter(tab =>
```
Comment (embbnux) [inline dev-server/containers/MainView/index.js]: you can set a tabs variable here and return in last with less codes

**Pull request merged**